devtools/create_project/cmake.cpp

```diff
@@ -174,6 +176,9 @@ if (TARGET SDL2::SDL2)
 endif()
 include_directories(${SDL2_INCLUDE_DIRS})
 
+# Explicitly support MacPorts (hopefully harmless on other platforms)
```

Comment:
Will this work in Windows platforms? There's a WIN32 define, perhaps it can be used for this as well?